src/jump_moi_overloads.jl

```diff
@@ -337,12 +347,12 @@ end
 
 # JuMP
 
-function reverse_differentiate!(model::JuMP.Model)
-    return reverse_differentiate!(JuMP.backend(model))
+function reverse_differentiate!(model::JuMP.Model; kwargs...)
```

Comment:
avoid kwargs if we can use MOI attributes

Reply:
changed to MOI.AbstractModelAttribute

Reply:
remove the kwargs now?